Testing/AutomatedA11yTesting.md

```diff
@@ -0,0 +1,79 @@
+# A11y Testing Recommendation
+
+The ability to automate even a fraction of our a11y testing will have huge benefits for IITB.
+
+While a11y test automation does not remove the need to manually test our applications it does allow our a11y assessors to focus on more valuable work.
+
+There are quite a few free or open source a11y tools out there that we can use to automate the testing of our work.
+
+The developer experience team reached out to some folks in the web a11y community in Ottawa and the GoC through Twitter [Calvin Rodo's thread link](https://twitter.com/CalvinR/status/1093265325041352722) and were provided with some tools to look into.
+
+## Proof of Concept
+
+The DevX Team wrote a proof of concept (not completed) using Puppeteer and aXe Core.
+
+The link to this proof of concept is below: <https://github.com/esdc-devx/auto-a11y-demo>
+
+## Tools Selected
```

Comment:
As it's part of the PoC lets add the header levels (not part of original document)
```suggestion
### Tools Selected
```